src/clinvar_streams/storage/database_sqlite/sink.clj

```diff
@@ -637,10 +637,64 @@
   (let [release-date (:release_date release-sentinel)
         get-simple (fn [table-name]
                      (let [s (format "select * from %s where dirty = 1 and release_date = ?" table-name)
-                           u (format "update %s set dirty = 0 where release_date = ?" table-name)
+                           ;u (format "update %s set dirty = 0 where release_date = ?" table-name)
                            rs (query @db-client/db [s release-date])]
                        ;(execute! @db-client/db [u release-date])
-                       rs))]
+                       rs))
+        get-variation (fn []
+                        (let [s (str "select v.* from variation v
+                                where release_date = ? and
+                                (dirty = 1 or
+                                 exists (
+                                   select * from gene_association ga
+                                   where ga.dirty = 1 and ga.variation_id = v.id))")
+                              dirty-variations (query @db-client/db [s release-date])
+                              _ (log/infof "Got %d dirty variations" (count dirty-variations))
+                              ; Add the latest gene_association entries for this variant
+                              ; if a gene association has been deleted most recently, don't include it
+                              dirty-variations
+                              (map (fn [variation]
+                                     (let [gene-association-sql (str "select * from gene_association ga "
+                                                                     "where ga.variation_id = ? "
+                                                                     "and ga.release_date = "
+                                                                     " (select max(release_date) "
+                                                                     "  from gene_association "
+                                                                     "  where event_type <> 'delete' and "
```

Comment:
Actually not sure this `<> 'delete'` filter is correct. We want to include gene associations on this variation which have not been deleted, but this may result in linking in a prior version of the gene association before it was deleted.

Reply:
Moved in https://github.com/clingen-data-model/clinvar-streams/pull/19/commits/c6f4ca2177a3bd453e3d5e89011459e1aa48bf0f

Reply:
@theferrit32 do you need me to identify some vcv that has had deleted gene associations over time so you can validate this?